### PR TITLE
fix: toast to snackbar for ticket and favourite fragment

### LINKED ISSUE #785: Remove toasts in favor of Snackbars
Toasts are kind of deprecated as any user who has disabled notifications for an app won't be able to see toast as well

Change toasts to snackbars wherever used

### PR DESCRIPTION
**Pull request by GOVINDDIXIT**
Fixes #785
Merge with #838

GIF for the change:
gif-190115_234944

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt [hunk 1]
```diff
@@ -146,4 +148,13 @@ class LoginFragment : Fragment() {
             else -> super.onOptionsItemSelected(item)
         }
     }
+
+    private fun showSnackbar() {
+        val textSnackbar = arguments?.getString("snackbar_message")
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt]: Where is this used below?

Comment (GOVINDDIXIT) [reply app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt]: I only used this to check whether snackbar should be showed or not

Comment (iamareebjamal) [reply app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt]: You should show the passed string. Imagine telling the fragment what needs to be shown as snackbar and it ignores your message and shows whatever it wants

Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt]: Extract the key to a contant as previously told

File: app/src/main/res/layout/fragment_tickets.xml [hunk 2]
```diff
@@ -8,119 +8,125 @@
...
-            android:text="@string/register"
-            android:textColor="@android:color/white" />
-    </LinearLayout>
-</androidx.core.widget.NestedScrollView>
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:id="@+id/ticketTableHeader"
+                android:background="@drawable/filled_border"
+                android:layout_marginTop="8dp"
```
Comment (nikit19) [inline app/src/main/res/layout/fragment_tickets.xml]: In dimens.xml

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -166,11 +166,15 @@
...
+    <string name="log_in_first">"You need to log in first!"</string>
+
     <!--welcome fragment-->
     <string name="start_text_1">Where</string>
     <string name="start_text_2">do you</string>
     <string name="start_text_3">go out?</string>
     <string name="start_text_4">We\'ll show you what\'s good near you</string>
     <string name="pick_a_city">Pick a city</string>
 
+
```
Comment (nikit19) [inline app/src/main/res/values/strings.xml]: Extra

File: app/src/main/res/layout/fragment_tickets.xml (hunk 2 above)
Comment (GOVINDDIXIT) [reply app/src/main/res/layout/fragment_tickets.xml]: @nikit19 Will change once back to machine

**Pull request merged**